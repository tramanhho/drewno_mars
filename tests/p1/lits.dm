32432 57493 83 5840935 
bdhsf nfdjksf lfnhsdf _mwoe_ ___ ___
"hello this is a test"