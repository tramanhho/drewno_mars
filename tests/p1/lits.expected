INTLITERAL:32432	[1,1]
INTLITERAL:57493	[1,7]
INTLITERAL:83	[1,13]
INTLITERAL:5840935	[1,16]
ID:bdhsf	[2,1]
ID:nfdjksf	[2,7]
ID:lfnhsdf	[2,15]
ID:_mwoe_	[2,23]
ID:___	[2,30]
ID:___	[2,34]
STRINGLITERAL:"hello this is a test"	[3,1]
EOF [3,23]